### ISSUE TITLE
Kane & Lynch: Dead Men (8080)

### ISSUE THREAD

**Issue by shoober420** [ValveSoftware/Proton#3534]
# Compatibility Report
- Name of the game with compatibility issues: Kane & Lynch: Dead Men
- Steam AppID of the game: https://steamdb.info/app/8080/

## System Information
- GPU: GTX 760
- Driver/LLVM version: NVIDIA 440.59
- Kernel version: 5.5.4
- https://github.com/shoober420/linux-scripts/blob/main/home/shoober420/sysinfonvidia
- Proton version: 5.0-2

LOG:
[steam-8080.log](https://github.com/ValveSoftware/Proton/files/4211334/steam-8080.log)

## Symptoms
Crash at launch, possibly related to "xlive.dll".

https://bugs.winehq.org/show_bug.cgi?id=23532

**Comment by kisak-valve** [comment]
Lines of interest from the log:
`err:module:import_dll Library xlive.dll (which is needed by L"Z:\\home\\jukilo420_-\\.local\\share\\Steam\\steamapps\\common\\Kane and Lynch Dead Men\\kaneandlynch.exe") not found`
`err:module:LdrInitializeThunk Importing dlls for L"Z:\\home\\jukilo420_-\\.local\\share\\Steam\\steamapps\\common\\Kane and Lynch Dead Men\\kaneandlynch.exe" failed, status c0000135`

**Comment by iWeaker** [comment]
1) Start the game once for proton to set up the WINEPREFIX
2) Open up a terminal and run command: protontricks 8080 --gui (considering that protontricks is installed)
3) Click "OK" on the warning about a 64-bit WINEPREFIX
3) Check "Enable silent install" and press "OK"
4) Stay on "Select the default wineprefix" and press "OK"
5) Check "Install a Windows DLL or component" and press "OK"
6) Check "gfw", "d3dx9_31", "d3dx9_34", press "OK" and wait for the operation to finish
7) Press "OK", "Cancel" to exit from protontricks and close the terminal
8) Set the following lauch command: PROTON_NO_ESYNC=1 PROTON_USE_WINED3D=1 PROTON_FORCE_LARGE_ADDRESS_AWARE=1 %command%
9) Play the game!

**Comment by frostworx** [comment]
Just fyi:
I recently added Kane & Lynch: Dead Men (8080) support to my [steamtinkerlaunch](https://github.com/frostworx/steamtinkerlaunch) which takes care that

- GFWL directories are removed,
- WLIDSvcM.exe is killed
- xliveless is installed into the gamedir (warns if not, and optionally installs it if files are provided - no automatic download)
- d3dx9_31 and d3dx9_34 are installed
- PROTON_NO_ESYNC and PROTON_NO_FSYNC are set to 1

TL;DR: the game works ootb with stl (when xliveless files are found)

**Comment by Simmonz** [comment]
I tried running Kane and Lynch: Dead Men but it just crashes at launch. I have attached a log file:

[steam-8080.log](https://github.com/ValveSoftware/Proton/files/9151682/steam-8080.log)

I have checked whether there are updates for my system available and I am up to date

System Information:

Distro: Manjaro 21.3.4

GPU: RX 580 8GB

Driver/LLVM version: Mesa 22.1.3/14.0.6

Kernel version: 5.18.12-3-MANJARO

Proton version: 7.0-3

https://gist.github.com/Simmonz/5f037a00d7e4d2ebdd7d8c923a6f00e2
